mobile/mobile_46.asm

```diff
@@ -559,7 +559,7 @@ Function1184a5:
 	dw Function1188c8
 	dw Function11878d
 	dw Function118903
-	dw Function118a8f
+	dw Set_ExchangeDownloadURL
```

Comment:
What is the "`Set_`" prefix? Generally an underscore separator would be for something that will apply to every item in this list, e.g. if it's a jumptable for news actions then `News_` or `NewsCmd_` might be appropriate. If it's just because this is the routine to set the exchange download URL, then `SetExchangeDownloadURL` is fine.

Reply:
It's just because it sets the url, so someone reading through the code has some idea what the jump table is for, removing the underscore makes sense